pyiron/vasp/interactive.py

```diff
@@ -56,8 +56,11 @@ def interactive_enforce_structure_reset(self, reset):
             "interactive_enforce_structure_reset() is not implemented!"
         )
 
-    def get_structure(self, iteration_step=-1):
-        return GenericInteractive.get_structure(self, iteration_step=iteration_step)
+    def get_structure(self, iteration_step=-1, wrap_atoms=True):
+        structure = GenericInteractive.get_structure(self, iteration_step=iteration_step, wrap_atoms=wrap_atoms)
+        if len(self.get_magnetic_moments()) > 0:
```

Comment:
Shouldn't it be like this?

```
try:
    if len(self.get_magnetic_moments()) > 0:
except NotImplementedError:
    pass
```

Reply:
Should it? Since the function is located at `vasp/interactive.py`, there won't be a `NotImplementedError`, will there?

Reply:
Oh it's inside VASP, then it's even worse, because that would mean `get_structure` behaves differently depending on the DFT code. It should be inside generic DFT.

Reply:
Right! the question is where to put it. At `dft/job/generic.py` or `atomistic/job/atomistic.py`?

Reply:
In `dft/job/generic.py` is something similar implemented already. I will change this.
https://github.com/pyiron/pyiron/blob/8625d1c003fb848dd4a8bf05514c3b40506537c7/pyiron/dft/job/generic.py#L125

Reply:
> 
> 
> Oh it's inside VASP, then it's even worse, because that would mean `get_structure` behaves differently depending on the DFT code. It should be inside generic DFT.

I quess we should discuss this in detail. Since the jobs use `get_structure()` of the `VaspInteractive` class, the one in `dft/job/generic.py` will never be called!

Reply:
I added now the changes @jan-janssen suggested at the meeting.